src/main/java/com/puppycrawl/tools/checkstyle/checks/metrics/ClassDataAbstractionCouplingCheck.java

```diff
@@ -23,7 +23,7 @@
 
 /**
  * <p>
- * Measures the number of instantiations of other classes
+ * Measures the number of distinct classes that are instantiated
```

Comment:
When reading the documentation, it wasn't immediately obvious that only distinct class instances are counted for ClassDataAbstractionCouplingCheck. While this behavior makes sense, I felt it could be clearer. Therefore, I took the liberty to update the documentation to make this point explicit, ensuring it's easier to understand for future readers.